backend/app/models/user.rb

```diff
@@ -9,8 +9,9 @@ class User < ApplicationRecord
   has_many :project_developers, through: :favourite_project_developers
   has_many :favourite_investors, dependent: :destroy
   has_many :investors, through: :favourite_investors
+  has_many :owner_accounts, class_name: "Account", foreign_key: "owner_id", dependent: :destroy
```

Comment:
Technically it's possible, but business logic should be `has_one` I think. I would also name it `owned_accounts` or `owned_account` if `has_one`.